daemon/driver.go

```diff
@@ -132,19 +132,24 @@ func (i *DriverInstance) Start() error {
 	return nil
 }
 
-func (i *DriverInstance) dial() error {
+func (i *DriverInstance) dial(ctx context.Context) error {
 	addr := filepath.Join(i.tmp, GRPCSocket)
 
 	opts := []grpc.DialOption{
 		grpc.WithDialer(func(addr string, t time.Duration) (net.Conn, error) {
 			return net.DialTimeout("unix", addr, t)
 		}),
+		// always wait for the connection to become active
 		grpc.WithBlock(),
-		grpc.WithTimeout(ConnectionTimeout),
+		// we want to know sooner rather than later
+		// TODO(dennwc): sometimes the initialization of the container takes >5 sec
+		//               meaning that the time between Container.Start and the actual
+		//               execution of a Go server (not the native driver) takes this long
```

Comment:
Just curious, could you please help me understand where would this time be visible in our current instrumentation?

Reply:
It will be included in the `bblfshd.pool.newDriverPool` span in Jaeger.

But there is no direct measurement for this specific delay - it will account for the whole "start driver, wait for the gRPC server" step. I plan to add more instrumentation later because it requires changes to SDK and drivers.